### PR TITLE
Gauss-Chebyshev quadrature of the third kind

### PR DESCRIPTION
**Pull request by susilehtola**
As discussed in #27

### CODE REVIEW

File: include/integratorxx/quadratures/gausscheby3.hpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      weights[i] = 2.0*pi_ov_2n_p_1 * points[i];
+
+      // However, since we want the rule with a unit weight factor, we
+      // divide the weights by sqrt((1-x)/x).
+      weights[i] /= std::sqrt((1.0 - points[i])/points[i]);
+    }
+
+    return std::make_tuple(points, weights);
+  }
+}
```
Comment (wavefunction91) [inline include/integratorxx/quadratures/gausscheby3.hpp:77]: Update per #30

**Review by wavefunction91 — CHANGES_REQUESTED** [review]

File: include/integratorxx/quadratures/gausscheby3.hpp (hunk 1 above)
Comment (susilehtola) [reply include/integratorxx/quadratures/gausscheby3.hpp]: Done, also converted from `[0,1]` to `[-1,1]` like the other rules

**Review by wavefunction91 — CHANGES_REQUESTED** [review]
Can you merge with `master` and add this new header to the includes in `test/1d_quadratures.cxx`? Just want to make sure everything compiles before it gets checked in, even if UTs aren't in place.

**Comment by susilehtola** [comment]
> Can you merge with `master` and add this new header to the includes in `test/1d_quadratures.cxx`? Just want to make sure everything compiles before it gets checked in, even if UTs aren't in place.

done

**Review by wavefunction91 — APPROVED** [review]

**Pull request merged**